library/src/scala/tasty/Reflection.scala

```diff
@@ -423,37 +423,41 @@ class Reflection(private[scala] val internal: CompilerInterface) { self =>
   def rootContext: Context = internal.rootContext // TODO: Should this be moved to QuoteContext?
   given Context = rootContext // TODO: Should be an implicit converion from QuoteContext to Context
 
-  extension ContextOps on (self: Context) {
-    /** Returns the owner of the context */
-    def owner: Symbol = internal.Context_owner(self)
+  given ContextOps as Context.type = Context
```

Comment:
This seems to suggest something is wrong with the language design.

Reply:
It is more about the automatically imported scopes for abstract types. With opaque types we look into the module but for these ones we are missing it. I plan on minimizing this limitation and ideally remove those givens later